hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileGetContentsTest.java

```diff
@@ -108,41 +107,29 @@ void validatesQueryWhenValidFile() throws Throwable {
 
         final var query = createGetFileContentQuery(fileId.fileNum());
         given(context.query()).willReturn(query);
-        given(context.createStore(ReadableFileStore.class)).willReturn(readableStore);
 
         assertThatCode(() -> subject.validate(context)).doesNotThrowAnyException();
     }
 
     @Test
     void validatesQueryIfInvalidFile() throws Throwable {
-        readableFileState.reset();
-        final var state = MapReadableKVState.<Long, File>builder("FILES").build();
-        given(readableStates.<Long, File>get(FILES)).willReturn(state);
-        final var store = new ReadableFileStoreImpl(readableStates);
-
-        final var query = createGetFileContentQuery(fileId.fileNum());
+        final var query = createGetFileContentQuery();
         when(context.query()).thenReturn(query);
-        when(context.createStore(ReadableFileStore.class)).thenReturn(store);
 
-        assertThatThrownBy(() -> subject.validate(context))
-                .isInstanceOf(PreCheckException.class)
-                .has(responseCode(ResponseCodeEnum.INVALID_FILE_ID));
+        assertThrowsPreCheck(() -> subject.validate(context), INVALID_FILE_ID);
     }
 
     @Test
-    void validatesQueryIfDeletedFile() throws Throwable {
+    void validatesQueryEvenWhenFileDeletedInState() throws Throwable {
         givenValidFile(true);
         readableFileState = readableFileState();
         given(readableStates.<FileID, File>get(FILES)).willReturn(readableFileState);
         readableStore = new ReadableFileStoreImpl(readableStates);
 
         final var query = createGetFileContentQuery(fileId.fileNum());
         when(context.query()).thenReturn(query);
-        when(context.createStore(ReadableFileStore.class)).thenReturn(readableStore);
 
-        assertThatThrownBy(() -> subject.validate(context))
-                .isInstanceOf(PreCheckException.class)
-                .has(responseCode(ResponseCodeEnum.FILE_DELETED));
+        assertDoesNotThrow(() -> subject.validate(context));
```

Comment:
Might be better here to assert the response attributes than just "the call does not throw"...